packages/eslint-config-eventbrite/rules/style.js

```diff
@@ -1,6 +1,16 @@
 // The rules ultimately override any rules defined in legacy/rules/style.js
 module.exports = {
     rules: {
+        // enforce dangling commas at the end of arrays, objects, imports & exports
+        // http://eslint.org/docs/rules/comma-dangle
+        'comma-dangle': ['error', {
+            'arrays': 'always-multiline',
+            'objects': 'always-multiline',
+            'imports': 'always-multiline',
+            'exports': 'always-multiline',
+            'functions': 'ignore',
+        }],
```

Comment:
🙌

Reply:
Yeah we for some reason didn't have dangling commas on imports which was always confusing...